repository/user_repository.go

```diff
@@ -41,3 +44,16 @@ func (repo *userRepo) FindUser(ctx context.Context, email string) (*domain.User,
 
 	return &user, nil
 }
+
+func (repo *userRepo) CreateUserQuery(ctx context.Context, name, email, password string, isAdmin bool) (*domain.User, error) {
```

Comment:
Also, don't implement the methods in this MR..
raise a separate MR for that

if there is an error in `go build`
then, just add dummy implementation.

for example:

```func (repo *userRepo) CreateUser(ctx context.Context, user domain.User) (*domain.User, error) {
return nil, nil
}```